src/components/MoneyRequestConfirmationList.js

```diff
@@ -92,13 +95,22 @@ const propTypes = {
 
     /** File source of the receipt */
     receiptSource: PropTypes.string,
+
+    /** Collection of categories attached to a policy */
+    policyCategories: PropTypes.objectOf(
```

Comment:
Looks like this is duplicated from what you created in `categoryPropTypes.js`. Can you just use ObjectOf categoryPropTypes?

Reply:
Done - b0da998742f4ba73d399dcb62471678ed38151de